contrib/99designs/gqlgen/tracer.go

```diff
@@ -0,0 +1,106 @@
+package gqlgen
+
+import (
+	"fmt"
+	"context"
+	"math"
+
+	"github.com/99designs/gqlgen/graphql"
```

Comment:
These imports go at the end :-)
stdlib, our packages, external packages with an empty line between each group